tests/phpunit/api/v3/OrderTest.php

```diff
@@ -171,4 +171,27 @@ public function addOrder($isPriceSet, $amount = 300, $extraParams = array()) {
     return $this->callAPISuccess('Contribution', 'create', $p);
   }
 
+  /**
+   * Test delete order api
+   */
+  public function testDeleteOrder() {
+    $order = $this->addOrder(FALSE, 100);
+    $params = array(
+      'contribution_id' => $order['id'],
+    );
+    try {
+      $this->callAPISuccess('order', 'delete', $params);
+      $this->fail("Missed expected exception");
+    }
+    catch (Exception $expected) {
+      $this->callAPISuccess('contribution', 'create', array(
```

Comment:
A trivial point - but we are trying to capitalise the entity as a convention & lower case the action - @colemanw  may have documented that somewhere?
ie
$this->callAPISuccess('Contribution', 'create', array(